routes/Favourite.js

```diff
@@ -95,4 +96,18 @@ router.post('/listFavourite', urlencodedParser, async(req, res) => {
     })
 })
 
+router.post('/getImgPath', urlencodedParser, async(req, res) => {
+    celebrity.Celebrity.findOne({
+        name : req.body.click
+    }, {
+        "imgPath" : 1
```

Comment:
what does imgPath 1 mean?

Reply:
get imgPath only, it won't return other attributes like gender, name etc except the id